lib/Doctrine/ORM/EntityManager.php

```diff
@@ -707,7 +707,12 @@ public function detach($entity)
      */
     public function merge($entity)
     {
-        @trigger_error('Method ' . __METHOD__ . '() is deprecated and will be removed in Doctrine ORM 3.0.', E_USER_DEPRECATED);
+        Deprecation::trigger(
+            'doctrine/orm',
+            '2.7',
+            'https://github.com/doctrine/orm/issues/8461',
+            'Method EntityManager::merge() is deprecated and will be removed in Doctrine ORM 3.0.'
```

Comment:
Why not use `__METHOD__` here and below?

Reply:
Yse, my bad :) Fixed